frontend/src/pages/publicProfile/index.tsx

```diff
@@ -28,8 +29,13 @@ export function makeUserProfileLinkProps(userProfileData: UserProfileLocationSta
 
 export default function PublicProfile() {
   const location = useLocation();
-  const { user, isShowFloatingButton = false } = location.state as UserProfileLocationState;
-  return user ? (
+  const state = location.state as UserProfileLocationState;
```

Comment:
can you try this:
`const {state} = useLocation<UserProfileLocationState>()`

Reply:
Dont think they support that 

<img width="371" alt="image" src="https://user-images.githubusercontent.com/24352004/198944636-6351a651-85dc-4ef1-9fc2-a92e31465f0d.png">

Reply:
![Uploading image.png…]()